policies.md

```diff
@@ -83,19 +83,19 @@ Sometimes serious grading errors are made, for example when a grader did not see
 
 Balancing these two sides is difficult. In MDS the policy is as follows: if a grade is challenged in a way that is deemed unreasonable, the student will receive a warning. This decision will be made by the instructor (not TA). If a student receives three warnings, the student will lose the privilege to challenge grades for the remainder of the program. Examples of unreasonable requests include extremely minor complaints (e.g. half a mark on an assignment) or repeatedly contesting the same issue once a decision has been reached. This policy applies to both labs and quizzes.
 
-## Quiz policies
+## Quiz Policies
 
-Unless otherwise specified, MDS quizzes are **closed book**, with allowance of a double-sided A4 cheatsheet. You may only access Canvas during the quiz.
+Unless otherwise specified, MDS quizzes are **closed** book, with the allowance of a digital double-sided A4 cheatsheet that must be uploaded to PrairieLearn before the quiz. You may only access PrairieLearn during the quiz.
 
-**It is NOT permitted to communicate with anyone else, including via posting quiz questions on internet forums, during a quiz.**
+During the quiz window, students are not allowed to disclose, discuss, or share any part of the quiz with any other individual, except as directly permitted or required by the course instructors. This includes discussions in person, online, or through any electronic means. Violation of this policy will result in academic penalties, which may include failure of the quiz or failure of the course.
 
 **Use of AI tools (e.g., ChatGPT) is also strictly prohibited and can lead to serious consequences.**
 
 The following are also **NOT** permitted:
 
 - taking a quiz from a remote location
 - leaving the quiz room early
```

Comment:
This may not be true anymore. I am confirming with Firas. We had this policy because everyone in the room wrote the same quiz with the same length. Last year, CBTF allowed students to leave the room after completing the exam.

Reply:
Cool, let me know when you have a confirmation.